README.md

```diff
@@ -4,11 +4,13 @@ This TypeScript SDK provides tools for working with Ethereum stealth addresses a
 
```

Comment:
Getting this error, any idea why?
<img width="1114" alt="image" src="https://github.com/ScopeLift/stealth-address-sdk/assets/61768337/05df37cb-8010-443d-bf22-0d96be09f1fc">

Reply:
Now passing.